### PR TITLE
Generalize DEV Digest

### PR DESCRIPTION
**Pull request by Ridhwana**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Generalize the Digest email further.

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
**The Config Update**
Screenshot 2020-06-01 at 13 24 49

**The Email Update**
screencapture-localhost-3000-rails-mailers-digest-mailer-digest-email-2020-06-01-13_25_39

## Added tests?

- [x] yes
- [ ] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

File: app/models/site_config.rb [hunk 1]
```diff
@@ -29,6 +29,7 @@ class SiteConfig < RailsSettings::Base
   field :community_description, type: :string, default: "A constructive and inclusive social network. Open source and radically transparent."
   field :community_member_description, type: :string, default: "amazing humans who code."
   field :community_member_label, type: :string, default: "user"
+  field :community_action, type: :string, default: "coding"
```
Comment (Ridhwana) [inline app/models/site_config.rb:32]: Leaving this as a default for now, all defaults will be removed for future communities in a subsequent PR based on [this issue](https://github.com/thepracticaldev/dev.to/projects/15#card-37093809)

File: app/views/internal/configs/show.html.erb [hunk 2]
```diff
@@ -170,6 +170,15 @@
               <div class="alert alert-info">Used to determine what a member will be called e.g developer, hobbyist etc.</div>
             </div>
 
+            <div class="form-group">
+              <%= f.label :community_action, "Community action" %>
```
Comment (Ridhwana) [inline app/views/internal/configs/show.html.erb:174]: Community Action sounded okay to me but I spent a lot of time on trying to get the appropriate name for the field, if anyone feels its terrible, let me know 😅

**Comment by Ridhwana** [comment]
@leewynne just pinging you on this PR :) You'll need to add a community_action field in your site config.

**Comment by leewynne** [comment]
Gotcha!

**Pull request merged**